sys/log/full/syscfg.yml

```diff
@@ -52,6 +52,11 @@ syscfg.defs:
             Turned off by default when CONSOLE_TICKS is on.
         value: 1
 
+    LOG_CONSOLE_PRETTY_TIMESTAMP_CLOCK:
+        description: >
+            Print timestamp as time yyyy-mm-ddThh:ss:mm.tttttttt
```

Comment:
`tttttt`

Reply:
changed from 8 to 6 characters, also `t` to `u` (as for micro seconds)